modules/ROOT/assets/attachments/sync-gateway-admin.yaml

```diff
@@ -188,7 +188,10 @@ parameters:
   open_revs:
     name: open_revs
     in: query
-    description: 'Option to fetch specified revisions of the document. The value can be `all` to fetch all leaf revisions or an array of revision numbers (i.e. open_revs=["rev1", "rev2"]). If this option is specified the response will be in multipart format. Use the `Accept: application/json` request header to get the result as a JSON object.'
+    description: |
+      Option to fetch specified revisions of the document. The value can be `all` to fetch all leaf revisions or an array of revision numbers (i.e. open_revs=["rev1", "rev2"]). Only [leaf revision](glossary.html) bodies are guaranteed to be returned.
```

Comment:
@tleyden Are you talking about tombstone leaf revisions that have been pruned?  Strictly speaking if they've been pruned they are no longer leaf revisions.

Reply:
Yeah it might be worth clarifying that point.  "Only leaf revision bodies that haven't been pruned are guaranteed to be returned".

Also, with XATTRS, isn't there another caveat around tombstoned leaf revisions that were deleted outside the metadata purge interval?